runtime/bcverify/rtverify.c

```diff
@@ -1359,6 +1410,16 @@ verifyBytecodes (J9BytecodeVerificationData * verifyData)
 					}
 					type = POP;
 				}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+				if (bc != JBwithfield) {
+					if ((type == BCV_BASE_TYPE_NULL) && (NULL != hashTableFind(valueTypesTable, &utf8string))) {
+						/* cannot set value type to null
+						 * TODO: proper exception */
+						printf("BIG ERROR\n");
+						goto _inconsistentStack;
```

Comment:
Other than an NLS message, how should I implement the exception throwing? I notice there are some constants defined for the `verboseErrorCode` in `j9consts.h`, but are these mapped to something?

Reply:
ya they used in `errormessageframeworkrtv.c`. 

I think in this case it would be better to use the `_miscError` target and add your own errorcode (similar to BCV_ERR_BAD_ACCESS_PROTECTED)